packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## `Pending Change`
```

Comment:
Changelog update isn't necessary here, since the change won't result in a new package version being published.

Reply:
@t1m0thyj I am guessing this means I can ignore the CHANGELOG.md test failing? Thanks for all the suggestions! I went ahead and made the changes and pushed them to the PR.